src/Libraries/Encryption/Cryptor.php

```diff
@@ -0,0 +1,112 @@
+<?php
+namespace Quantum\Libraries\Encryption;
+
+class Cryptor
+{
+    private $res;
+    private $digest_alg = 'sha512';
+    private $private_key_bits = 1024;
+    private $private_key_type = OPENSSL_KEYTYPE_RSA;
+    private $config = [];
+    private $keys = [];
+    private $encryptedData;
+    private $decryptedData;
+    public static $cipher = 'AES-256-CBC';
+
+    public function __construct()
+    {
+        $this->set_config();
+        $this->set_res();
+        $this->set_priv_key();
+    }
+
+    public function set_config()
+    {
+        $this->config = [
+            "digest_alg" => $this->digest_alg,
+            "private_key_type" => $this->private_key_type,
+            "private_key_bits" => $this->private_key_bits
+        ];
+    }
+
+    private function set_res()
+    {
+        $this->res = openssl_pkey_new($this->config);
+    }
+
+    private function set_priv_key()
+    {
+        openssl_pkey_export($this->res, $privKey);
+        $pubKey = openssl_pkey_get_details($this->res);
+
+        $this->keys = [
+            'public_key' => base64_encode($pubKey["key"]),
+            'private_key' =>  base64_encode($privKey),
+        ];
+    }
+
+    public function get_keys($obj = true)
+    {
+        return $obj ? (object) $this->keys : $this->keys;
+    }
+
+    private function public_encrypt($data, $pubKey)
+    {
+        openssl_public_encrypt($data, $this->encryptedData, $pubKey);
+    }
+
+    public function encrypt_data($data, $pubKey)
+    {
+        $this->public_encrypt($data, base64_decode($pubKey));
+        return base64_encode($this->encryptedData);
+    }
+
+    private function private_decrypt($encrypted_data, $privKey)
+    {
+        openssl_private_decrypt($encrypted_data, $this->decryptedData, $privKey);
+    }
+
+    public function decrypt_data($encrypted_data, $privKey)
+    {
+        $this->private_decrypt(base64_decode($encrypted_data), base64_decode($privKey));
+        return $this->decryptedData;
+    }
+
+    private static function encrypt($plaintext)
```

Comment:
Make this method public and let this be the one instead of encryptString (remove the encryptString method)